### ISSUE TITLE
Gandi webacc delete --backend throws an error with --port option

### ISSUE THREAD

**Issue by themouette** [Gandi/gandi.cli#130]
When executing the following:

```
gandi webacc delete --backend 2001:4b98:dc2:43:216:3eff:fe2c:xxxx --port 80
```

I encountered the following error:

```
Traceback (most recent call last):
  File "/home/julien/Projects/gandi/15-ans/venv/bin/gandi", line 9, in <module>
    load_entry_point('gandi.cli==0.12', 'console_scripts', 'gandi')()
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/gandi.cli-0.12-py2.7.egg/gandi/cli/__main__.py", line 8, in main
    cli(obj={})
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/click-3.3-py2.7.egg/click/core.py", line 610, in __call__
    return self.main(*args, **kwargs)
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/click-3.3-py2.7.egg/click/core.py", line 590, in main
    rv = self.invoke(ctx)
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/gandi.cli-0.12-py2.7.egg/gandi/cli/core/cli.py", line 168, in invoke
    click.Group.invoke(self, ctx)
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/click-3.3-py2.7.egg/click/core.py", line 936, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/click-3.3-py2.7.egg/click/core.py", line 782, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/click-3.3-py2.7.egg/click/core.py", line 416, in invoke
    return callback(*args, **kwargs)
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/click-3.3-py2.7.egg/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/click-3.3-py2.7.egg/click/core.py", line 416, in invoke
    return callback(*args, **kwargs)
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/gandi.cli-0.12-py2.7.egg/gandi/cli/commands/webacc.py", line 183, in delete
    result = gandi.webacc.backend_remove(backend)
  File "/home/julien/Projects/gandi/15-ans/venv/local/lib/python2.7/site-packages/gandi.cli-0.12-py2.7.egg/gandi/cli/modules/webacc.py", line 131, in backend_remove
    (backend['ip'], backend['port']))
KeyError: 'port'
```

**Comment by sayoun** [comment]
Will be fixed in next 0.14 release

**Comment by themouette** [comment]
great!

thank you

**Comment by sayoun** [comment]
Fixed in release 0.14

**Comment by themouette** [comment]
Great !

Le mer. 13 mai 2015 18:41, Dejan Filipovic <notifications@github.com> a
écrit :
